packages/sfui/frameworks/nuxt/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@storefront-ui/nuxt",
```

Comment:
Just wanted to raise one question - shouldn't we call it @storefront-ui/nuxt-module? `nuxt` seems a bit too generic - but maybe it's only me

Reply:
The general approach that I was thinking of was the convention for these meta-framework specific themes would be `@storefront-ui/{meta-framework}/{framework}`.

So like:
- `@storefront-ui/nuxt`
- `@storefront-ui/astro/vue`
- `@storefront-ui/astro/react`

Specifically for this Nuxt module, I think having `nuxt` is the normal thing I've seen since this is what ends up in some's `nuxt.config.ts` modules.

So like 
```
export default defineNuxtConfig({
  modules: ['@storefront-ui/nuxt'],
})
```

Reply:
Okay, just wanted to make sure that there won't be anything wrong with naming package like this, thanks!

Regarding astro example:
[Official NPM package name validator](https://www.npmjs.com/package/validate-npm-package-name) says we cannot name the package `@storefront/astro/vue`.
<img width="576" alt="image" src="https://user-images.githubusercontent.com/10456649/229372039-bb0979d1-adf4-4d8c-810b-d408af425f1f.png">
So, if we would like to allow users to import things from `@storefront-ui/astro/react`and `@storefront-ui/astro/vue`  we would need to export these from a single package - `@storefront-ui/astro`. And I would not mix packages for different frameworks, it might lead to troubles with build and other things.